### PR TITLE
FIX: Dragzone overlapping

### LINKED ISSUE #1294: Overlapping of the dragzone with the image in import image module
**Please Note: We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more**
### What's the problem (or idea)
The dragzone is overlapping the image in the import module
Screenshot from 2019-11-09 17-10-47

### What should be done?
We can add a new class  to the following div element
https://github.com/publiclab/image-sequencer/blob/cff82c998f351907e724a7151670885745100066/src/modules/ImportImage/Ui.js#L11
and it's corresponding style in 
https://github.com/publiclab/image-sequencer/blob/cff82c998f351907e724a7151670885745100066/examples/demo.css#L62
The final result should be 
Screenshot from 2019-11-14 00-42-49

### Where to look
https://beta.sequencer.publiclab.org/examples/#steps=import-image{}

### PR DESCRIPTION
**Pull request by VladimirMikulic**
Changes:
- inline styles moved from ../ImportImage/Ui.js to demo.css
- minor code improvements

The change was necessary because in the old UI drag zone was overlapping
an image.

test

Resolves #1294

@harshithpabbati @keshav234156 requesting review :)

### CODE REVIEW

File: examples/demo.css [hunk 1]
```diff
@@ -69,12 +69,14 @@ body > .container-fluid {
...
-.hover {
-  background: #eee;
+.import-image-zone {
+  margin: 10px auto 30px auto;
+  max-width: 230px;
+  min-width: 230px;
 }
 
-.dropzone input {
-  max-width: 100%;
```
Comment (keshav234156) [inline examples/demo.css]: Can we just keep the max-width: 100 %. If yes, then also change max-width  of image-import-zone accordingly

**Comment by VladimirMikulic** [comment]
@keshav234156 
I kept max-width: 100%; rule and added max-width: fit-content; to .import-image-zone.
fit-content is necessary because on 100% text is squished in three dots(...).

Everything else remained the same :)

**Comment by keshav234156** [comment]
@VladimirMikulic As the text was getting squashed with 100% that why  I suggested you to increase the max-width in import-image-zone.Can you please do the changes.

**Comment by VladimirMikulic** [comment]
@keshav234156 as you've requested, here is the change.

**Comment by keshav234156** [comment]
@VladimirMikulic don't increase max-width in .import-image-zone input ie keep it 100%.Change max-width in import-image-zone from 230 to 250.Hope it's clear now ,If not please let me know.

**Comment by VladimirMikulic** [comment]
I apologise. Apparently, I got lost with all these widths 😅.
Here is the new look:

new

File: examples/demo.css [hunk 2]
```diff
@@ -69,14 +69,24 @@ body > .container-fluid {
...
 }
 
+.import-image-zone {
+  margin: 10px auto 30px auto;
+  max-width: 250px;
+  min-width: 230px;
+}
+
+.import-image-zone input {
+  max-width: fit-content;
```
Comment (keshav234156) [inline examples/demo.css]: please change this as well to max-width: 100%;

Comment (VladimirMikulic) [reply examples/demo.css]: If you want the box to look like this, I can do that :)

prob

Comment (VladimirMikulic) [reply examples/demo.css]: That's why .import-image-zone input needs to be:
```css
.import-image-zone input {
    max-width: fit-content;
}
```
or
```css
.import-image-zone input {
    max-width: 101%;
}
```

Comment (VladimirMikulic) [reply examples/demo.css]: That 1% less(ie. 100%) squishes text. I suggest that we keep it at fit-content.

Comment (keshav234156) [reply examples/demo.css]: it won' squash if you keep max-width :250 px in .import-image-zone and 100% .import-image-zone input .I have already tried it locally.Can you also give one more try.

Comment (keshav234156) [reply examples/demo.css]: .import-image-zone {
  margin: 10px auto 30px auto;
  max-width: 250px;
  min-width: 230px;
}

.import-image-zone input {
  max-width: 100%;
}

**Comment by VladimirMikulic** [comment]
That should do it.

new2

**Comment by keshav234156** [comment]
Looks absolutely perfect to me now.Thanks for your patience.

**Comment by VladimirMikulic** [comment]
@keshav234156 thank you for your clarification and patience. We did it together.

**Comment by keshav234156** [comment]
@jywarren can you please review the changes.

**Comment by VladimirMikulic** [comment]
@jywarren can we get this merged? I would like you to review the changes :)

**Comment by jywarren** [comment]
Looks great! Thank you! Let's merge after https://github.com/publiclab/image-sequencer/pull/1325; we'll update the branch just after that merges!

**Comment by jywarren** [comment]
Awesome!

**Pull request merged**